dask/multiprocessing.py

```diff
@@ -206,8 +232,8 @@ def get(
     try:
         # Run
         result = get_async(
-            pool.apply_async,
-            len(pool._pool),
+            partial(multiprocessing_apply_async, pool),
+            pool._max_workers,
```

Comment:
In theory, [this should work]( https://github.com/python/cpython/blob/a041e116db5f1e78222cbf2c22aae96457372680/Lib/concurrent/futures/process.py#L588-L601 ). However it doesn't seem to work in practice.